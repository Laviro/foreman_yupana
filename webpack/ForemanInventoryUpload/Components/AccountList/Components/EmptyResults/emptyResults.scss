.blank-slate-pf {
  margin-top: 100px;

  .no_results_icon {
    margin-right: 5px;
  }
}
