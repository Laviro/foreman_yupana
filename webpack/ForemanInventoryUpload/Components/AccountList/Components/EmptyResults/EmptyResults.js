import React from 'react';
import { EmptyState, Icon } from 'patternfly-react';
import { translate as __ } from 'foremanReact/common/I18n';
import './emptyResults.scss';

const inventoryEmptyResults = () => (
  <EmptyState>
    <EmptyState.Title>
      <Icon className="no_results_icon" name="meh-o" />
      {__("Oops! Couldn't find organization that matches your query")}
    </EmptyState.Title>
  </EmptyState>
);

export default inventoryEmptyResults;
