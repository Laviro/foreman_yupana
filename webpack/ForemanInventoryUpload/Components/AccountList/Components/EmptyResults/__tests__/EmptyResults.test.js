import { testComponentSnapshotsWithFixtures } from 'react-redux-test-utils';

import EmptyResults from '../EmptyResults';

const fixtures = {
  'render without Props': {},
  /** fixtures, props for the component */
};

describe('EmptyResults', () => {
  describe('rendering', () =>
    testComponentSnapshotsWithFixtures(EmptyResults, fixtures));
});
