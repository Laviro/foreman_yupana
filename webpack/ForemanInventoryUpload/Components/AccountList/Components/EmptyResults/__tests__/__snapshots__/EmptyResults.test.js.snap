// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyResults rendering render without Props 1`] = `
<EmptyState
  className=""
>
  <EmptyStateTitle
    className=""
  >
    <Icon
      className="no_results_icon"
      name="meh-o"
      type="fa"
    />
    Oops! Couldn't find organization that matches your query
  </EmptyStateTitle>
</EmptyState>
`;
